docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -0,0 +1,62 @@
+---
+description: Integrate Codacy with your Git workflow to display analysis results and code coverage as status checks on your pull requests and optionally block merging pull requests.
+---
+
+# Integrating Codacy with your Git workflow
+
+Once you've [configured your repository](configuring-your-repository.md) to best match your use case, integrate Codacy with your Git workflow to display analysis results and code coverage as status checks on your pull requests.
+
+In particular, you can configure quality gates to block merging pull requests that don't meet the quality standards of your team. This ensures the quality of the changes to your codebase, preventing the introduction of security issues and untested code.
+
+To integrate Codacy with your Git workflow, follow these steps:
+
+1.  [Configuring the quality gate rules](#configuring-gate)
+1.  [Activating the Git provider integration](#git-provider-integration)
+1.  [Blocking merging pull requests](#blocking-pull-requests) (optional)
+
+## 1. Configuring the quality gate rules {: id="configuring-gate"}
```

Comment:
The configuration of the quality gate rules is only necessary for the integration with the Git workflow? Isn't it necessary besides this integration?

Reply:
To enforce quality gates, you must also integrate Codacy with the Git provider. Moreover, the Git provider only blocks PRs if the quality gates are set. So this step and the following one are very correlated.

You can also set quality gates with no integration, and this will display warnings on the Codacy UI, but it won't block the development workflow.